advocacy_docs/edb-postgres-ai/console/using/index.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Using the console
+indexCards: simple
+description: Using the EDB Postgres AI console to from creating clusters and managing users to connecting cloud services and enabling encryption
```

Comment:
This sentence needs review. For example, "to from".

Reply:
```suggestion
description: Using the EDB Postgres AI console from creating clusters and managing users to connecting cloud services and enabling encryption.
```